### PR TITLE
support projects in image commands

### LINKED ISSUE #2950: Support projects in platform images
Part of https://github.com/neuro-inc/no-code/issues/77

### PR DESCRIPTION
**Pull request by zubenkoivan**
closes #2950

### CODE REVIEW

File: neuro-cli/src/neuro_cli/disks.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ async def ls(
             uri_fmtr: URIFormatter = str
         else:
             uri_fmtr = uri_formatter(
-                username=root.client.username,
+                project_name=root.client.username,
```
Comment (dalazx) [inline neuro-cli/src/neuro_cli/disks.py:57]: same question here.

File: neuro-cli/src/neuro_cli/blob_storage.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ async def lsbucket(
             uri_fmtr: URIFormatter = str
         else:
             uri_fmtr = uri_formatter(
-                username=root.client.username,
+                project_name=root.client.username,
```
[the remark this answers is not in the document]
Comment (zubenkoivan) [reply neuro-cli/src/neuro_cli/blob_storage.py:99]: this should be handled in a separate PR for buckets, here I just renamed parameter which doesn't break anything

File: neuro-cli/src/neuro_cli/disks.py (hunk 1 above)
Comment (zubenkoivan) [reply neuro-cli/src/neuro_cli/disks.py:57]: the same here, just renaming

**Review by dalazx — APPROVED** [review]
👍

**Pull request merged**